### ISSUE TITLE
Cannot leave fullscreen mode

### ISSUE THREAD

**Issue by ashinzekene** [Automattic/simplenote-electron#739]
#### Steps to reproduce
1. Install app on Linux mint mate

#### What I expected
To be able to get out of fullscreen mode 

#### What happened instead
Cannot get out of fullscreen mode

#### OS version
Linux Mint Mate 18

#### Steps to reproduce
1. Install app on Linux mint mate

#### What I expected
To be able to get out of fullscreen mode
I should be able to leave fullscreen from the menu below

#### What happened instead
Cannot get out of fullscreen mode

#### OS version
Linux Mint Mate 18

#### Screenshot / Video
![image](https://user-images.githubusercontent.com/20991583/38219064-f3a23d22-36cb-11e8-946c-29aa46066061.png)

**Comment by ashinzekene** [comment]
Was finally able to leave full screen. I read through the code :weary:  :weary:  :weary:

**Comment by mirka** [comment]
@ashinzekene Did you find out what was wrong? Let us know if you think there was a bug or a usability issue.

**Comment by ashinzekene** [comment]
Except a user already knows the shortcut, he can't leave full screen. There should be an option to toggle fullscreen here 
![image](https://user-images.githubusercontent.com/20991583/45914931-fb1f0300-be43-11e8-91c8-abba922ddaa2.png)

**Comment by mirka** [comment]
Ok, I see how this might be a usability issue on Linux. On Windows the menu bar is always present, and on Mac the menu bar appears when you hover the cursor. There is no way out on Linux if you don't know the <kbd>F11</kbd> shortcut (which is standard, but still...)

We might be able to add a [fullscreen toggle button](https://fontawesome.com/icons/expand-alt?style=light) somewhere in the top toolbar, if/when we do a slight redesign of the controls.

**Comment by ashinzekene** [comment]
Yeah, I actually had to read through the source code to be able to leave full screen

**Comment by mirka** [comment]
I guess it would be enough if the Full Screen mode would reset after quitting and relaunching the app. Most apps seem to behave that way.

**Comment by ashinzekene** [comment]
No problem with that then

**Comment by rakhi2104** [comment]
@mirka will making the app launch in windowed mode rather than fullscreen mode fix this ? or should we add a fullscreen toggle ?

**Comment by mirka** [comment]
@rakhi2104 

> should we add a fullscreen toggle ?

No need, a fullscreen toggle is already in the View menu.

**Comment by rakhi2104** [comment]
Fine :+1:
